features/0434-outofband/README.md

```diff
@@ -119,6 +123,9 @@ The items in the message are:
 - `label` - [optional] a self-attested string that the receiver may want to display to the user, likely about who sent the out-of-band message.
 - `goal_code` - [optional] a self-attested code the receiver may want to display to the user or use in automatically deciding what to do with the out-of-band message.
 - `goal` - [optional] a self-attested string that the receiver may want to display to the user about the context-specific goal of the out-of-band message.
+- `accept` - [optional] an array of mime types in the order of preference of the sender that the receiver can use in responding to the message. 
+If `accept` is not specified, the receiver uses its preferred choice to respond to the message (sender accepts any).
```

Comment:
I like this idea of this when an OOB message is being used. How would we use this with 0-RTT when only one type is supported though?

For example, we're directly sending a plaintext VC request template for our first message (over a TLS channel) and only supporting didcommV2. I think this wouldn't end up solving interop concerns with our implementation though because we opt to lead with a 0-RTT message rather than an OOB message which would be used to specify this.

Reply:
@kdenhartog I think this concept carries over to other similar scenarios. When the first message is not DIDComm, a header should be used to indicate what the sender accepts. This PR generally enables the `accept` property (regardless of transport) for those using OOB. If you are not using OOB, you could use an equivalent for your situation (e.g., the `accept` HTTP header).

Reply:
Also note - I used the `accept` property name in this PR based on the `accept` HTTP header.